### ISSUE TITLE
Add wait for storage zone replication

### ISSUE THREAD

**Issue by dmitriyshashkin** [ayeressian/bunnycdn-storage-deploy#47]
BunnyCDN allows storage replication. It may take some time for all the replicas to sync. Storage API responds before the replicas are synchronized. Which means that if we update a file and immediately after that we clear the cache, the edge servers that are connected to replica can re-cache the old version of the file:

1. We update the file
2. We clear the cache
3. The edge server gets a request
4. Edge server gets old version of the file from the storage replica and caches it again
5. Storage replica gets updates from the main storage

In the end the edge server keeps responding with the outdated file, despite the cache was purged.

Bunny support has previously confirmed the existence of such behavior, their suggestion was to add 5 seconds wait between the update of the file and the purging of the cache.

**Comment by ayeressian** [comment]
Hi @dmitriyshashkin sorry for the late response. I was busy with personal matters. Does BunnyCDN have this behavior documented?

**Comment by dmitriyshashkin** [comment]
I don't think so. But the opposite behaviour is not documented as well. There is simply no mention of the storage zone replication in the docs: https://docs.bunny.net/reference/put_-storagezonename-path-filename

**Comment by dmitriyshashkin** [comment]
Correct.
